tests/cpp/test_multidevice_matmul.cpp

```diff
@@ -253,8 +253,8 @@ TEST_F(DistributedMatmulTest, LayoutNT_ReduceScatter) {
   TensorView* ab = mul(a_b, b_b); // (Ko,M,N,Ki)
   TensorView* c0 = sum(ab, {-1}); // (Ko,M,N,r)
   c0 = segment_set(c0);
-  std::vector<int64_t> orig_size = {K, M, N};
-  std::vector<int64_t> new_size = {K, Mo, Mi, N};
+  std::vector<int64_t> orig_size = {Ko, M, N};
+  std::vector<int64_t> new_size = {Ko, Mo, Mi, N};
```

Comment:
This fixes a trivial bug found in https://github.com/NVIDIA/Fuser/pull/2386#discussion_r1635591610, but I'm happy to separate it out if you prefer.